sql/slave.cc

```diff
@@ -7220,7 +7220,31 @@ event(errno: %d  cur_log->error: %d)",
     sql_print_error("Error reading relay log event: %s", errmsg);
   DBUG_RETURN(0);
 }
+#ifdef WITH_WSREP
+Log_event *wsrep_peak_event(rpl_group_info *rgi, ulonglong* event_size)
```

Comment:
wsrep_peak_event is used to compare event code type, so i would rather return that instead of complete object and rename it to wsrep_peak_event_code_type(...).
If this is could be used in some context, than it's fine.

Reply:
makes sense, I was not sure if only type is needed in caller side, but appears so